docs/en/compute/Tensorflow.md

```diff
@@ -35,22 +35,22 @@ and allow Tensorflow applications to access the data through Alluxio POSIX API.
 
 Run the following command to install FUSE on Linux:
 
-```
+```shell
```

Comment:
we have the annotation `console` other than `shell` in other docs, not quite sure which is the convention

Reply:
In my experience, I find that `console` colors text blocks inconsistently. It sometimes leaves line italicized and greyed out when it is not necessary. I find that `shell` does a better job with readability and coloring consistency, but I can change it back if `console` is the convention.